### PR TITLE
speed ups to rna quant ingest

### LINKED ISSUE #1549: Improve performance of rnaseq2ga module
We have developed new code that implements the same functionality as rnaseq2ga.rnaseq2ga() but is MUCH faster. This issue asks that the new faster version replace the old version without changing any of the parameters. @saupchurch

### PR DESCRIPTION
**Pull request by ejacox**
I made the expression id an auto increment field and sped up reading the expression data from the file. Closes #1549

### CODE REVIEW

**Comment by ejacox** [comment]
Added fix to oidc problem also to close #1566.

**Comment by ejacox** [comment]
@david4096 I removed uuid id generation, which was very slow. The auto increment was just for convenience. The ids can be generated by incrementing a counter in the code, too. On reflection, this will need to be changed to assign ids incrementally by RnaQunatifiction in order to have reproducible ids. The primary key will need to be changed to be the id column and the rna_quantification_id.

**Comment by ejacox** [comment]
I changed the id as described above and added an ingest test to increase the code coverage.

File: ga4gh/server/repo/rnaseq2ga.py [hunk 1]
```diff
@@ -157,6 +190,7 @@ def writeExpression(self, rnaQuantificationId, quantfilename,
                               rawCount, score, units, confidenceLow,
                               confidenceHi)
                 self._db.addExpression(datafields)
+                expressionId += 1
```
Comment (david4096) [inline ga4gh/server/repo/rnaseq2ga.py]: Cool! Do you think this same idea will work for adding expressions in parallel (ignoring sqlite)?

File: tests/datadriven/test_rna_quantification.py [hunk 2]
```diff
@@ -175,3 +180,22 @@ def testSearchExpressionLevelsWithFeatureIds(self):
...
+        """
+        tempDir = tempfile.mkdtemp(prefix="ga4gh_rna_quant",
+                                   dir=tempfile.gettempdir())
+        dbName = os.path.join(tempDir, "rnaQuantDB")
+        storeDb = rnaseq2ga.RnaSqliteStore(dbName)
+        storeDb.createTables()
+
+        testTsvFile = os.path.join(
+                            paths.testDataDir,
+                            "datasets/dataset1/rnaQuant/rsem_test_data.tsv")
```
Comment (david4096) [inline tests/datadriven/test_rna_quantification.py]: This test is really just guaranteeing the whole this doesn't go belly up, which is a great improvement!

File: ga4gh/server/repo/rnaseq2ga.py (hunk 1 above)
Comment (ejacox) [inline ga4gh/server/repo/rnaseq2ga.py]: Yes, since each file will be processed within it's own process/thread.

**Pull request merged**